turbinia/evidence.py

```diff
@@ -257,6 +257,15 @@ def __init__(self, plaso_version=None, *args, **kwargs):
     super(PlasoFile, self).__init__(*args, **kwargs)
 
 
+class PlasoCsvFile(PlasoFile):
+  """Psort output file evidence.  """
+
+  def __init__(self, plaso_version=None, *args, **kwargs):
+    """Initialization for Plaso File evidence."""
+    self.plaso_version = plaso_version
```

Comment:
Where do you get plaso_version from?

Reply:
Right now I don't, but there's a TODO in the plaso task code, :)

Reply:
ack